### PR TITLE
release: update manifest and helm charts for v0.0.17

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:
- Updates manifests and charts for release `v0.0.17`

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml [hunk 1]
```diff
@@ -21,15 +23,6 @@ spec:
             - --v=5
             - "--csi-address=unix://C:\\csi\\csi.sock"
             - --kubelet-registration-path={{ .Values.windows.kubeletRootDir }}\plugins\csi-secrets-store\csi.sock
-          lifecycle:
-              preStop:
```
Comment (tam7t) [inline charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml:25]: Just checking, are removing these hooks intentional?

**Review by tam7t — APPROVED** [review]

File: charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml (hunk 1 above)
Comment (aramase) [reply charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml:25]: @tam7t that's correct. The hooks were removed as part of https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/366. 

The latest `node-driver-registrar` images are distroless so these prestop commands don't work. The `node-driver-registrar` cleans up the registration socket as part of container shutdown.

Comment (tam7t) [reply charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml:25]: Thanks SGTM

**Pull request merged**